ghost/email-service/lib/email-service.js

```diff
@@ -73,7 +79,15 @@ class EmailService {
 
         // Check host limit for disabled emails or going over emails limit
         if (this.#limitService.isLimited('emails')) {
-            await this.#limitService.errorIfWouldGoOverLimit('emails');
+            await this.#limitService.errorIfWouldGoOverLimit('emails', {addedCount});
+        }
+
+        // Check email verification required
+        // We need to check this inside the job again
```

Comment:
It's a bit of an odd comment here because it's the same one as "outside" the job. Maybe worth rephrasing it in a way that explains the check here is an extra one to run the check on per-job bases - separate to the one done before scheduling jobs?

Reply:
You are right, this comment doesn't make sense 🤦🏻‍♂️